.github/workflows/build.yml

```
@@ -0,0 +1,49 @@
+name: Production build
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: 'Use Node.js ${{ matrix.node-version }}'
+        uses: actions/setup-node@v1
+        with:
+          node-version: '${{ matrix.node-version }}'
+
+      - name: Download all modules
+        run: npm i
```

Run `ci` instead of `i` to ensure clean dependencies
```suggestion
        run: npm ci
```